packages/components/tests/dummy/app/templates/components/dropdown.hbs

```diff
@@ -12,6 +12,56 @@
 
 <section>
   <h3 class="dummy-h3" id="section-component-api">Component API</h3>
+  <p class="dummy-paragraph">The
+    <code class="dummy-code">Dropdown</code>
+    component is composed by different parts, with their own APIs:</p>
+  <ul>
+    <li class="dummy-paragraph">
+      a main "container" (the dropdown itself)
```

Comment:
[question] I thought we didn't want to make this immediately obvious and that the disclosure component was going to only be available for design system component use? 

If this is not true, then we need to have a team conversation. 

If this is still true, then it does not need to be in this documentation.

Reply:
Can you explain? I don't understant. The `Dropdown` itself needs to be made available. Are you meaning the `Disclosure`?